calcom/cal.com
# refactor: <BookingListItem/>  to use <Link/> to navigate instead of r…

## What does this PR do?

Refactor the Booking List table to use the Link to navigate instead of using router.

Fixes #11591

## Type of change




## How should this be tested?

- Navigate to the Bookings Page.
- Click on any of the bookings available.
- The booking details should be visible.

## Mandatory Tasks
